www/docs/server/procedures.md

```diff
@@ -5,336 +5,88 @@ sidebar_label: Define Procedures
 slug: /server/procedures
 ---
 
-Procedures in tRPC are very flexible primitives to create backend functions; they use a builder pattern which means you can create reusable base procedures for different parts of your backend application.
+In tRPC a Procedure is ultimately just a function which you want to expose to your client. A Procedure can be either:
+
+- a `Query` - used to fetch data, generally does not change any data
+- a `Mutation` - used to send data, often for create/update/delete purposes
```

Comment:
third bullet point for subscriptions?

Reply:
Fair point, I think best for now just to point at the deocs though (we do need to bring those docs into the fold as Further isn't a great home for a feature)